johnzon-mapper/src/main/java/org/apache/johnzon/mapper/MapperBuilder.java

```diff
@@ -236,7 +238,8 @@ public Mapper build() {
                         supportEnumContainerDeserialization,
                         typeLoader, discriminatorMapper, discriminator,
                         deserializationPredicate, serializationPredicate,
-                        enumConverterFactory),
+                        enumConverterFactory,
+                        new Snippet(snippetMaxLength, generatorFactory)),
```

Comment:
we must be able to run with any JSON-P impl so we likely need some indirection there (fine to not be optimum if we don't run with johnzon-core but a generic json-p impl)

Reply:
I think this might have been a comment for the constructor as it referenced JsonGeneratorFactoryImpl.  I updated it back to Json.createGeneratorFactory.  If that wasn't what you meant, I sent an invite to the repo so you have access to do any adjustments you'd like.